_docs/quickstarts/scanner/descriptions.md

```diff
@@ -29,6 +29,22 @@ The bigquery scanner depends on the following Forseti Inventory pipelines:
 
 - `load_bigquery_datasets_pipeline`
 
+## Blacklist scanner
+
+VM instances with external IP addresses communicate with the outside world 
+and in case they were compromised or misbehaved, they could end up in various 
+blacklists and is known to be malicious i.e. sending spam, hosting C&C etc.
+The blacklist scanner audits all your VM instances in your environment and 
+determines if any VMs with external IP addresses are on a specific blacklist 
+you've configured.
+
+For example of how to define scanner rules see the [blacklist_rules.yaml](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/rules/blacklist_rules.yaml)
+rule file.
+
+Blacklist scanner depends on the following Forseti Inventory pipelines:
+
+- `load_instances_pipeline
```

Comment:
You forgot your closing ` and leading space in front of the '-'

Reply:
Done.